demisto_sdk/commands/common/constants.py

```diff
@@ -221,21 +219,344 @@ def no_common_server_python(path):
+               f'file {file_path} or run \'demisto-sdk format -p {file_path}\''
+
+    @staticmethod
+    def no_test_playbook(file_path, file_type):
+        return f'TP101 - You don\'t have a TestPlaybook for {file_type} {file_path}. ' \
+               f'If you have a TestPlaybook for this {file_type}, ' \
+               f'please edit the yml file and add the TestPlaybook under the \'tests\' key. ' \
+               f'If you don\'t want to create a TestPlaybook for this {file_type}, ' \
+               f'edit the yml file and add  \ntests:\n -  No tests\n lines to it or ' \
+               f'run \'demisto-sdk format -i {file_path}\''
+
+    @staticmethod
+    def test_not_in_conf_json(file_id):
+        return f"CJ101 - You've failed to add the {file_id} to conf.json"
+
+    @staticmethod
+    def remove_field_from_dashboard(file_path, field):
+        return f'DA100 - {file_path}: the field {field} needs to be removed.\n'
+
+    @staticmethod
+    def remove_field_from_widget(field, widget):
+        return f'WD100 - The field {field} needs to be removed from the widget: {widget}.\n'
+
+    @staticmethod
+    def include_field_in_dashboard(file_path, field):
+        return f'DA101 - {file_path}: the field {field} needs to be included. Please add it.\n'
+
+    @staticmethod
+    def include_field_in_widget(field, widget_name):
+        return f'WD101 - The field {field} needs to be included in the widget: {widget_name}. Please add it.\n'
+
+    @staticmethod
+    def description_missing_in_beta_integration(package_path):
+        return f"DS100 - No detailed description file was found in the package {package_path}. Please add one, " \
+               f"and make sure it includes the beta disclaimer note." \
+               f"It should contain the string in constant\"BETA_INTEGRATION_DISCLAIMER\""
+
+    @staticmethod
+    def no_beta_disclaimer_in_description(package_path):
+        return f"DS101 - Detailed description in beta integration package {package_path} " \
+               f"dose not contain the beta disclaimer note. It should contain the string in constant" \
+               f"\"BETA_INTEGRATION_DISCLAIMER\"."
+
+    @staticmethod
+    def no_beta_disclaimer_in_yml(file_path):
+        return f"DS102 - Detailed description field in beta integration {file_path} " \
+               f"dose not contain the beta disclaimer note. It should contain the string in constant" \
+               f" \"BETA_INTEGRATION_DISCLAIMER\"."
+
+    @staticmethod
+    def description_in_package_and_yml(package_path):
+        return f"DS103 - A description was found both in the " \
+               f"package and in the yml, please update the package {package_path}."
+
+    @staticmethod
+    def default_docker_error():
+        return 'DO100 - The current docker image in the yml file is the default one: demisto/python:1.3-alpine,\n' \
+               'Please create or use another docker image\n'
```

Comment:
Maybe we should add the option to skip this check if they really want to use it?

Reply:
isnt that what the ignore should do?